src/layout/parser/splits.rs

```diff
@@ -155,11 +155,16 @@ where
                             });
                         }
                     })
+                } else if tag.name() == b"SplitTimesAccuracy" {
+                    accuracy(reader, tag.into_buf(), |v| settings.split_time_accuracy = v)
```

Comment:
Also set it for the segment times here, as there is no such distinction in the original LS, so you would use this value there as well to specify the segment times accuracy.